crates/wallet/src/wallet/mod.rs

```diff
@@ -454,143 +403,88 @@ impl Wallet {
         })
     }
 
-    /// Either loads [`Wallet`] from the given [`ChangeSet`] or initializes it if one does not exist.
+    /// Loads [`Wallet`] from the previously persisted [`ChangeSet`] and descriptors.
+    ///
+    /// This method will fail if the given descriptors don't match those included in the
+    /// [`ChangeSet`].
     ///
-    /// This method will fail if the loaded [`ChangeSet`] has different parameters to those provided.
+    /// If the provided descriptors contain private keys the signers for those keys will be added
+    /// to the `Wallet`.
```

Comment:
```suggestion
    /// to the [`Wallet`].
```

Why not doclinking this one?

Reply:
Good catches on the docs, fixed.